cmd/config-v10.go

```diff
@@ -46,6 +46,7 @@ type aliasConfigV10 struct {
 	Path         string `json:"path"`
 	License      string `json:"license,omitempty"`
 	APIKey       string `json:"apiKey,omitempty"`
+	TrailingHdrs bool   `json:"trailingHeaders,omitempty"`
```

Comment:
maybe 'chunksChecksum' is a better name ?

Reply:
@klauspost why do we need this to be an `mc` config option? can this not be a `flag` just for uploads? such as `mc mirror` and `mc cp` ?

Reply:
@harshavardhana It is a server property whether it supports it or not, similar to v4 signatures.

It will *allow* various calls to use trailing headers, but it isn't tied to PutObject(and friends) as such.

Reply:
okay so it is not limited to uploads' alone? @klauspost or is it something that you want to keep it for future?